src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py

```diff
@@ -293,10 +265,39 @@ def timeseries(
         """
         legends = legend_manager.create_legends(num_legend_threshold, full_legends)
         for legend in legends:
-            p.add_layout(legend, 'right')
-        p.legend.click_policy = 'hide'
+            fig.add_layout(legend, 'right')
+        fig.legend.click_policy = 'hide'
+
+        return fig
+
+    def _init_figure(
+        self,
+        title: str,
+        ywheel_zoom: bool,
+        xaxis_type: str,
+        yaxis_label: Optional[str] = None,
```

Comment:
Variable names do not conform to the Snake Case rule.

```suggestion
        y_wheel_zoom: bool,
        x_axis_type: str,
        y_axis_label: Optional[str] = None,
```

Reply:
@rokamu623 
Thanks for your review.
But, I wanted to be consistent with the naming of the Bokeh library, so I only changed `yaxis_label` (fixed in https://github.com/tier4/CARET_analyze/pull/256/commits/77dffc103f9f959cabef1759243381fa8c525e81).
- [ywheel_zoom](https://docs.bokeh.org/en/latest/docs/user_guide/tools.html#wheelzoomtool)
- [xaxis](https://docs.bokeh.org/en/2.4.2/docs/reference/models/axes.html)